usr/src/cmd/zfs/zfs_main.c

```diff
@@ -3892,7 +3896,7 @@ zfs_do_send(int argc, char **argv)
 static int
 zfs_do_receive(int argc, char **argv)
 {
-	int c, err;
+	int c, err = 0;
```

Comment:
This change is harmless, but if the compiler is complaining about it, that seems like a compiler bug, because we always assign err before using it (AFAICT).